### PR TITLE
Create RunIssueSentinel.yml

### PR DESCRIPTION
**Pull request by megharai1302**
Added a file to run Sentinel tool in office-js repo

### CODE REVIEW

File: .github/workflows/RunIssueSentinel.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  Issue:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Issue Sentinel
+        uses: Azure/issue-sentinel@v1
+        with:
+          password: ${{secrets.ISSUE_SENTINEL_PASSWORD}}
```
Comment (anjalitp) [inline .github/workflows/RunIssueSentinel.yml:15]: Make sure secret has been stored with correct name

**Review by anjalitp — APPROVED** [review]

File: .github/workflows/RunIssueSentinel.yml (hunk 1 above)
Comment (megharai1302) [reply .github/workflows/RunIssueSentinel.yml:15]: Sure, I made the changes @anjalitp

**Comment by megharai1302** [comment]
> @megharai1302 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by megharai1302** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**